controllers/apply/under10k/fields.js

```diff
@@ -1466,9 +1466,7 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                 en: 'This person has to live in the UK.',
                 cy: 'Rhaid i’r person hwn fyw ym Mhrydain',
             }),
-            schema: Joi.fullName()
-                .compare(Joi.ref('mainContactName'))
```

Comment:
This constraint already applied for plain references but not seems to apply to references in extensions too. Looks like we can only have the error on one of the two fields. Annoying but not the end of the world.